tutorials/setup.md

```diff
@@ -17,28 +17,84 @@ Latest version of `runc` is expected to be installed on the system. It is picked
 
 **Required**
 
-Fedora, CentOS, RHEL, and related distributions:
+Fedora, RHEL<=7, CentOS and related distributions:
 
 ```bash
 yum install -y \
-  btrfs-progs-devel \
-  containers-common \
-  device-mapper-devel \
-  git \
-  glib2-devel \
-  glibc-devel \
-  glibc-static \
-  go \
-  golang-github-cpuguy83-go-md2man \
-  gpgme-devel \
-  libassuan-devel \
-  libgpg-error-devel \
-  libseccomp-devel \
-  libselinux-devel \
-  pkgconfig \
-  runc
+btrfs-progs-devel \
+containers-common \
+device-mapper-devel \
+git \
+glib2-devel \
+glibc-devel \
+glibc-static \
+go \
+golang-github-cpuguy83-go-md2man \
+gpgme-devel \
+libassuan-devel \
+libgpg-error-devel \
+libseccomp-devel \
+libselinux-devel \
+pkgconfig \
+runc
 ```
 
+
+RHEL 8 distributions:
+
+Make sure you are subscribed to the following repositories: \
+BaseOS/x86_64 \
+Appstream/x86_64
+
+Follow the guide below to subscribe to the repositories if not already subscribed: \
+https://access.redhat.com/solutions/265523
+
+
+This requires go version 1.12 or greater. Use the link below to install the appropriate go version for the system:\
+(Replace apt-get with yum for RHEL based machines) \
```

Comment:
```suggestion
(Replace apt-get with yum for RHEL based machines)
```

Reply:
That character is for the new line(formatting). I think that isn't causing a problem.

Reply:
Yes, but it seems to be visible in the rendered markdown.

Reply:
When we look at it on the github page, it does not shows up. I will check that again and get back to you.

Reply:
Here's is a link on how that looks! 
Let me know if any changes are required.
https://github.com/phariyan/cri-o/blob/master/tutorials/setup.md

Reply:
Sure, I will change that. Thanks!